include/webserv.hpp

```diff
@@ -37,23 +45,7 @@ std::string readHtml(const std::string &filePath);
  * in a standard HTTP response format, including headers for content type and length.
  * @return An HTTP response string including headers and the HTML content of the home page.
  */
-// std::string handleHomePage();
-
-/**
- * Handles the execution of a CGI script and captures its output to construct an HTTP response.
- * This function sets up a pipe for capturing the standard output of the CGI script, forks the process,
- * and executes the script in the child process using the provided environment settings. The parent process
- * reads the script's output from the pipe, waits for the script to finish execution, and then constructs
- * an HTTP response with the script's output. If an error occurs during the process, a 500 Internal Server Error
- * response is generated instead.
- *
- * @param argv An array of char pointers representing the arguments to be passed to the CGI script,
- *             including the script path as the first argument.
- * @param env An Environment object containing the environment variables to be passed to the CGI script.
- * @return A string representing the HTTP response generated from the CGI script's output, or a 500 Internal
- *         Server Error message if the script execution fails.
- */
-// std::string handleCGIRequest(const char *argv[], Environment env);
+std::string handleHomePage();
```

Comment:
Should this not be a method of the static response handler or be removed if not used?

Reply:
I think this can have its linear issue, it is in a way related to the way we serve errors, and to wether we are expecting an index.html in a directory :
https://linear.app/web-serv/issue/WEB-135/decide-of-a-custom-error-for-isdirectory-andand-havenoindexhtml

Reply:
Done: https://linear.app/web-serv/issue/WEB-136/decide-on-the-index-of-our-webserv